ddp-workspace/projects/ddp-brain/src/styles/activities.scss

```diff
@@ -463,6 +463,10 @@ select:disabled {
     }
 }
 
+.ddp-textarea {
```

Comment:
Is there a `overflow-y:scroll` in play somewhere here?

Reply:
Did not see one, but text area appears to be doing the right thing (or at least, what I would want it to do: stretch out to 10 lines and then show scrollbar if more text.
I doubt 10 lines would ever be needed (even in mobile it is a lot). But seems like reasonable behavior.